tests/ro-crate-metadata.json

```diff
@@ -0,0 +1,457 @@
+            "@type": [
+                "File",
+                "SoftwareSourceCode",
+                "ComputationalWorkflow"
+            ],
+            "attachment": [
+                {
+                    "@id": "exe/fastqc.cwl"
+                },
+                {
+                    "@id": "exe/trimmomatic_pe.cwl"
+                }
+            ],
+            "description": {
+                "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/README.md"
+            },
+            "input": [
+                {
+                    "@id": "#fastq_1"
+                },
+                {
+                    "@id": "#fastq_2"
+                }
+            ],
+            "name": "CWL_trimming_and_qc",
+            "output": [
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.1P.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.1U.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.2P.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.2U.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small_fastqc.html"
+                },
+                {
+                    "@id": "#ERR034597_2.small_fastqc.html"
+                },
+                {
+                    "@id": "stdout.log"
+                },
+                {
+                    "@id": "stderr.log"
+                }
+            ],
+            "programmingLanguage": {
+                "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl"
+            },
+            "version": "1.0.0",
+            "yevisId": "c13b6e27-a4ee-426f-8bdb-8cf5c4310bad"
+        },
+        {
+            "@id": "https://w3id.org/cwl/vv1.0/",
+            "@type": [
+                "WebPage"
+            ]
+        },
+        {
+            "@id": "https://www.commonwl.org/",
+            "@type": [
+                "WebPage"
+            ]
+        },
+        {
+            "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl",
+            "@type": "ComputerLanguage",
+            "alternateName": "CWL",
+            "identifier": {
+                "@id": "https://w3id.org/cwl/vv1.0/"
+            },
+            "name": "Common Workflow Language",
+            "url": {
+                "@id": "https://www.commonwl.org/"
+            },
+            "version": "v1.0"
+        },
+        {
+            "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/README.md",
+            "@type": [
+                "WebPage"
+            ],
+            "name": "README.md"
+        },
+        {
+            "@id": "exe/fastqc.cwl",
+            "@type": [
+                "File",
+                "FormalParameter",
+                "WorkflowAttachment",
```

Comment:
"WorkflowAttachment" is not in the context. If this is a subworkflow, the type should be `["File", "SoftwareSourceCode", "ComputationalWorkflow"]`

Reply:
technically It is not a subworkflow (which is defined here https://www.commonwl.org/v1.2/Workflow.html#SubworkflowFeatureRequirement) but it is just a secondary file that the primary workflow depends on (tool definition referred by workflow definition for instance). I think this is a leftover from the previous version which we added extra classes. I will fix it.